packages/replayer/src/replayer.ts

```diff
@@ -271,3 +281,15 @@ const shouldHoldBrowserOpen = () => {
     (process.env["METICULOUS_HOLD_BROWSER_OPEN"] ?? "").toLowerCase() === "true"
   );
 };
+
+const serializeError = (
+  error: unknown
+): { message: string | null; stack: string | null } => {
+  if (error == null) {
+    return { message: null, stack: null };
+  }
+  return {
+    message: (error as any).message ?? (error as any).toString?.() ?? null,
+    stack: (error as any).stack ?? null,
+  };
+};
```

Comment:
This one could be unsafe if the object thrown is not an `Error`, e.g. (`{ message: number }`).

```suggestion
const serializeError = (
  error: unknown
): { message: string | null; stack: string | null } => {
  if (error == null) {
    return { message: null, stack: null };
  }
  const message =
    toStringOrNull((error as any).message) ?? toStringOrNull(error);
  const stack = toStringOrNull((error as any).stack);
  return {
    message,
    stack,
  };
};

const toStringOrNull = (value: unknown): string | null =>
  value != null ? `${value}` : null;
```